app/src/main/java/com/example/testapp/data/Login.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.testapp.model
+package com.example.testapp.data
 
 class Login (val EmailAddress: String, var Password: String){
```

Comment:
Please add Java documentation at class level and function level with parameter description